support-frontend/assets/helpers/redux/checkout/product/contributionsSideEffects.ts

```diff
@@ -42,13 +42,13 @@ export function addProductSideEffects(
 				return;
 			}
 
-			const inThreeTierVariant = inThreeTierV2Variant(
+			const inThreeTier = showThreeTierCheckout(
```

Comment:
`inThreeTier` is only used/referenced in 1 place, maybe we can remove the variable and just call `showThreeTierCheckout` in the 1 place `inThreeTier` is used? It's a small simplification, but means less code / less variable names to consider.

Reply:
Updated